### DISCUSSION TITLE
Use UE and gNB(using two USRP) to transmit files

### DISCUSSION THREAD

**Discussion by peihaoY** [srsran/srsRAN_Project#453] [Ideas & Suggestions]
How can I use scp or other tools to transmit files so that I can know the evaluation of the RAN?

**Comment by brendan-mcauliffe** [comment]
Hi @peihaoY, 

You can send files using scp. We do not have a tutorial on this, but you should be able to find some info on using scp online. Another way to test would be to use an iPerf server/client which is shown in many of our tutorials. 

You will need an end-to-end set-up as is shown in [this tutorial](https://docs.srsran.com/projects/project/en/latest/tutorials/source/srsUE/source/index.html#over-the-air-setup ). You can also see how to use iPerf here.
